phpBB/includes/mcp/mcp_main.php

```diff
@@ -1455,6 +1455,26 @@ function mcp_fork_topic($topic_ids)
 						$counter[$row['poster_id']] = 1;
 					}
 				}
+
+				/**
+				* Modify the forked post's sql array before inserted into the database.
```

Comment:
I am not a native speaker but...

Reply:
I fixed this in the commit message, I'll fix it here too.